unlock-app/src/components/interface/locks/List/elements/LockCard.tsx

```diff
@@ -0,0 +1,182 @@
+              <div className="w-32 h-4 animate-pulse bg-slate-200"></div>
+              <div className="w-4 h-4 animate-pulse bg-slate-200"></div>
+              <div className="w-4 h-4 animate-pulse bg-slate-200"></div>
+            </div>
+          </div>
+        </div>
+        <div className="flex col-span-3 gap-14">
+          <DetailPlaceholder />
+          <DetailPlaceholder />
+          <DetailPlaceholder />
+        </div>
+        <div className="col-span-1">
+          <div className="w-6 h-6 animate-pulse bg-slate-200"></div>
+        </div>
+      </div>
+    </div>
+  )
+}
+
+export const LockCard = ({ lock, network, isLoading }: LockCardProps) => {
+  const web3Service = useWeb3Service()
+  const { networks } = useConfig()
+  const lockAddress = lock.address
+  const [isCopied, setCopied] = useClipboard(lockAddress, {
+    successDuration: 2000,
+  })
+
+  const { explorer } = networks?.[network] ?? {}
+
+  const explorerUrl = explorer?.urls?.address(lockAddress) || '#'
+
+  useEffect(() => {
+    if (!isCopied) return
+    ToastHelper.success(`Lock address copied`)
+  }, [isCopied])
+
+  const getSymbol = async () => {
+    return web3Service.getTokenSymbol(lockAddress, network)
+  }
+
+  const { data: symbol } = useQuery(
+    ['getSymbol', lockAddress, network],
+    async () => getSymbol()
+  )
+  const lockUrl = `/locks/lock?address=${lockAddress}&network=${network}`
+
+  if (isLoading) return <LockCardPlaceholder />
+
+  const duration =
+    lock.expirationDuration === UNLIMITED_KEYS_DURATION ? (
+      'Unlimited'
+    ) : (
+      <Duration seconds={lock.expirationDuration} />
+    )
+
+  return (
+    <>
+      <div className="px-12 py-4 bg-white shadow-lg rounded-2xl">
+        <div className="grid items-center justify-between grid-flow-col col-span-7 gap-4">
+          <div className="flex col-span-3 gap-3">
+            <div className="bg-gray-200 rounded-full h-14 w-14"></div>
+            <div className="flex flex-col gap-2">
+              <span className="text-2xl font-bold">{lock.name}</span>
+              <div className="flex items-center gap-3">
+                <span>{addressMinify(lockAddress)}</span>
+                <Button
+                  variant="transparent"
+                  className="p-0 m-0"
+                  onClick={setCopied}
+                >
+                  <CopyIcon size={20} />
+                </Button>
+                <a href={explorerUrl} target="_blank" rel="noreferrer">
+                  <Button variant="transparent" className="p-0 m-0">
+                    <ExternalLinkIcon
+                      size={20}
+                      className="text-brand-ui-primary"
+                    />
+                  </Button>
+                </a>
+              </div>
+            </div>
+          </div>
+          <div className="flex col-span-3 gap-14">
+            <Detail
+              label="Price"
+              value={lock.keyPrice}
```

Comment:
I think it would be useful to show the currency's symbol here

Reply:
symbol is already present

https://github.com/unlock-protocol/unlock/pull/9680/files#diff-647b2cdb24847dcc5e0a4da903aaa355aa8b3f13c6cb3a845248d7dc31bef38dR158